label_studio/ml_models/models.py

```diff
@@ -39,6 +45,8 @@ class ModelInterface(models.Model):
         'organizations.Organization', on_delete=models.CASCADE, related_name='model_interfaces', null=True
     )
 
+    skill_name = models.CharField(max_length=255, choices=SkillNames.choices, default=SkillNames.TEXT_CLASSIFICATION)
```

Comment:
`default=` is unsafe if we need to roll back; until we get on django 5 and can use `db_default`, every new field needs to be nullable and have no default set.

Reply:
changed

Reply:
(default value is set in the serializers in LSE now, would like to remove it in future but that'd be too big of a change surface area for this PR, especially FE)